desktop/ui/src/main/java/org/datacleaner/widgets/SelectionListener.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.widgets;
+
+public interface SelectionListener<N> {
+    void onSelected(FilenameTextField filenameTextField, N selected);
```

Comment:
Maybe we should get rid of that instead of renaming it. FileSelectionListener is the only interface that implements SelectionListener<N>. It is exactly how it was before this PR...